tests/unit/s2n_client_signature_algorithms_extension_test.c

```diff
@@ -90,26 +89,19 @@ int main(int argc, char **argv)
             GUARD(s2n_stuffer_write_uint16(&signature_algorithms_extension, sig_hash_algs.iana_list[i]));
         }
 
-        struct s2n_array *parsed_extensions = s2n_array_new(sizeof(struct s2n_client_hello_parsed_extension));
-        struct s2n_client_hello_parsed_extension *parsed_named_group_extension = NULL;
-        EXPECT_OK(s2n_array_pushback(parsed_extensions, (void **)&parsed_named_group_extension));
-        parsed_named_group_extension->extension_type = TLS_EXTENSION_SIGNATURE_ALGORITHMS;
-        parsed_named_group_extension->extension = signature_algorithms_extension.blob;
-
         /* If only unknown algorithms are offered, expect choosing a scheme to fail for TLS1.3 */
         conn->actual_protocol_version = S2N_TLS13;
-        EXPECT_SUCCESS(s2n_client_extensions_recv(conn, parsed_extensions));
+        EXPECT_SUCCESS(s2n_client_signature_algorithms_extension.recv(conn, &signature_algorithms_extension));
```

Comment:
nit: not related to your changes but could be more specific in the comment, 
` /* If only unknown signature algorithms are offered, expect choosing a signature scheme to fail for TLS1.3 */`

Reply:
Arguably "signature" is implied by the context of the test :) Without a good reason, I'd rather not confuse the diff by updating a comment for a test I didn't change the behavior of.